app/templates/components/ui-table/cell/events/view/speakers/cell-simple-sessions.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="ui unordered list">
+  {{#each record as |session|}}
```

Comment:
you can't refer to a related record like this, @ritikamotwani . use 
```hbs
 {{#each record.sessions as |session|}}
```
. As `sessions` are a related field.